eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -353,6 +354,41 @@ void choosesTransformerFactoryInConcurrentEnvironment() {
         }
     }
 
+    @Test
```

Comment:
@maxonfjvipon Maybe it's better to place that tests in `ParseMojoTest`?

Reply:
@volodya-lombrozo I don't think so. We execute `OptimizeMojo` here and check if it fails. Parsing on this step completes successfully.